### PR TITLE
docs: add registration part to collator docs

### PR DESCRIPTION
**Pull request by alexand3rwilke**
## Summary
+ Added t0rn faucet
+ Added some necessary steps to register_as_candidate 
+ Changed version to match the latest one

### CODE REVIEW

File: docs/t0rn-collator.md [hunk 1]
```diff
@@ -113,8 +113,36 @@ The Aura key must be inserted into the keystore *after* startup:
...
-Your collator should be running and also produce blocks eventually!
+## Get some t0rn from faucet 
 
+Your Collator needs some funds to register on testnet.
+
+Go to the [t0rn testnet faucet](https://dev.net.t3rn.io/faucet/), insert your substrate address and get some t0rn to cover transaction costs.
+
+## Register as a candidate
+
+1. Go to [polkadot.js](https://polkadot.js.org/apps/#/explorer), switch to t0rn and connect your collator account using polkadot.js
```
Comment (chiefbiiko) [inline docs/t0rn-collator.md:124]:
```suggestion
1. Go to the [polkadot.js app](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fdev.net.t3rn.io#/accounts) and connect your collator account by clicking "Add account", then inserting your previously generated secret phrase aka mnemonic.
```

File: docs/t0rn-collator.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 # Run t3rn Collator on Rococo testnet (t0rn)
```
Comment (chiefbiiko) [inline docs/t0rn-collator.md:1]:
```suggestion
# Run a t0rn Testnet Collator
```

File: docs/t0rn-collator.md [hunk 3]
```diff
@@ -113,8 +113,36 @@ The Aura key must be inserted into the keystore *after* startup:
...
 
+Your Collator needs some funds to register on testnet.
+
+Go to the [t0rn testnet faucet](https://dev.net.t3rn.io/faucet/), insert your substrate address and get some t0rn to cover transaction costs.
+
+## Register as a candidate
+
+1. Go to [polkadot.js](https://polkadot.js.org/apps/#/explorer), switch to t0rn and connect your collator account using polkadot.js
+
+2. Set your session keys with your connected collator account under
```
Comment (chiefbiiko) [inline docs/t0rn-collator.md:126]:
```suggestion
2. Generate a new session key pair and obtain the corresponding public key. Copy it to your clipboard as it is needed in the next step.
```

File: docs/t0rn-collator.md [hunk 4]
```diff
@@ -113,8 +113,36 @@ The Aura key must be inserted into the keystore *after* startup:
...
+
+```
+Developer/RPC calls/author --> rotateKeys
```
Comment (chiefbiiko) [inline docs/t0rn-collator.md:129]:
```suggestion
Developer --> RPC calls --> author --> rotateKeys()
```

File: docs/t0rn-collator.md [hunk 5]
```diff
@@ -113,8 +113,36 @@ The Aura key must be inserted into the keystore *after* startup:
   --key-type aura
 ```
 
-Your collator should be running and also produce blocks eventually!
+## Get some t0rn from faucet
```
Comment (chiefbiiko) [inline docs/t0rn-collator.md:116]:
```suggestion
## Get Some T0RN Balance
```

File: docs/t0rn-collator.md [hunk 6]
```diff
@@ -113,8 +113,36 @@ The Aura key must be inserted into the keystore *after* startup:
   --key-type aura
 ```
 
-Your collator should be running and also produce blocks eventually!
+## Get some t0rn from faucet 
 
+Your Collator needs some funds to register on testnet.
+
+Go to the [t0rn testnet faucet](https://dev.net.t3rn.io/faucet/), insert your substrate address and get some t0rn to cover transaction costs.
```
Comment (chiefbiiko) [inline docs/t0rn-collator.md:120]:
```suggestion
Go to the [t0rn testnet faucet](https://dev.net.t3rn.io/faucet/), insert your substrate address and get some T0RN to cover transaction costs.
```

File: docs/t0rn-collator.md [hunk 7]
```diff
@@ -113,8 +113,36 @@ The Aura key must be inserted into the keystore *after* startup:
...
+3. Set the Session Key for your collator under
+
+```
+Developer/Extrinsics --> session -> setKeys(sr25519_pubkey, 0x1234)
+```
+
+4. Now finally register your collator as candidate.
+
+```
+Developer/Extrinsics --> collatorSelection -> registerAsCandidate
```
Comment (chiefbiiko) [inline docs/t0rn-collator.md:141]:
```suggestion
Developer --> Extrinsics --> collatorSelection -> registerAsCandidate()
```

File: docs/t0rn-collator.md [hunk 8]
```diff
@@ -113,8 +113,36 @@ The Aura key must be inserted into the keystore *after* startup:
...
+2. Set your session keys with your connected collator account under
+
+```
+Developer/RPC calls/author --> rotateKeys
+```
+
+3. Set the Session Key for your collator under
+
+```
+Developer/Extrinsics --> session -> setKeys(sr25519_pubkey, 0x1234)
```
Comment (chiefbiiko) [inline docs/t0rn-collator.md:135]:
```suggestion
Developer --> Extrinsics --> session -> setKeys(sr25519_pubkey, 0x1234)
```

File: docs/t0rn-collator.md [hunk 9]
```diff
@@ -113,8 +113,36 @@ The Aura key must be inserted into the keystore *after* startup:
...
+
+1. Go to [polkadot.js](https://polkadot.js.org/apps/#/explorer), switch to t0rn and connect your collator account using polkadot.js
+
+2. Set your session keys with your connected collator account under
+
+```
+Developer/RPC calls/author --> rotateKeys
+```
+
+3. Set the Session Key for your collator under
```
Comment (chiefbiiko) [inline docs/t0rn-collator.md:132]:
```suggestion
3. Set the session key for your collator under
```

File: docs/t0rn-collator.md [hunk 10]
```diff
@@ -1,7 +1,7 @@
 # Run t3rn Collator on Rococo testnet (t0rn)
 
 
-This guide outlines the essential minimum of steps required to run a collator for t0rn - a release candidate of t3rn to Rococo testnet. Guide uses the `v1.0.0-rc.2` release to Rococo, please however always use the latest available version.
+This guide outlines the essential minimum of steps required to run a collator for t0rn - a release candidate of t3rn to Rococo testnet. Guide uses the `v1.1.0-rc.0` release to Rococo, please however always use the latest available version.
```
Comment (chiefbiiko) [inline docs/t0rn-collator.md:4]:
```suggestion
This guide outlines the essential minimum of steps required to run a collator for t0rn - a release candidate of t3rn on the Rococo testnet. This guide uses the `v1.1.0-rc.0` release, however always use the latest available version.
```

**Review by chiefbiiko — CHANGES_REQUESTED** [review]
Thanks for elaborating this guide - all my suggestions are just spelling things.

**Review by MaciejBaj — DISMISSED** [review]

**Comment by MaciejBaj** [comment]
Good to go after resolving @chiefbiiko's points

**Review by chiefbiiko — APPROVED** [review]
LGTM

**Review by MaciejBaj — APPROVED** [review]

**Pull request merged**